File: java/tokens.java
import org.openapitools.client.ApiClient;
import org.openapitools.client.ApiException;
import org.openapitools.client.Configuration;
import org.openapitools.client.auth.*;
import org.openapitools.client.model.*;
import org.openapitools.client.api.TokenApi;
import java.util.ArrayList;
import java.util.List;

public class Tokens {
    public static void main(String[] args) {
        String accessToken = "";

        ApiClient defaultClient = Configuration.getDefaultApiClient();
        defaultClient.setBasePath("http://localhost:3000/api/v1");
        defaultClient.addDefaultHeader("Authorization", "Bearer " + accessToken);

        TokenApi apiInstance = new TokenApi(defaultClient);
        try {
            IssueTokenRequest issueTokenRequest = new IssueTokenRequest();
            issueTokenRequest.setAmount(10000);
            issueTokenRequest.setTokenType(1);
            TokenResponse issueTokenResponse = apiInstance.issueToken(issueTokenRequest);
            System.out.println(issueTokenResponse);

            String reissueTokenId = "c14a1faf8089c2e2843438caa85cfc943b47d26a1e4dd5b279c72b4b4da4d67161";
            ReissueTokenRequest reissueTokenRequest = new ReissueTokenRequest();
            reissueTokenRequest.setAmount(1000);
            TokenResponse reissueTokenResponse = apiInstance.reissueToken(reissueTokenId, reissueTokenRequest);
            System.out.println(reissueTokenResponse);

            String transferTokenId = "c14a1faf8089c2e2843438caa85cfc943b47d26a1e4dd5b279c72b4b4da4d67161";
            TransferTokenRequest transferTokenRequest = new TransferTokenRequest();
            transferTokenRequest.setAmount(10000);
            transferTokenRequest.setAddress("mvZy7U661vyRwk2BHnZ1c5vBxg6jvQAhKK");
            TokenResponse transferTokenResponse = apiInstance.transferToken(transferTokenId, transferTokenRequest);
            System.out.println(transferTokenResponse);

            Boolean confirmationOnly = true;
            List<GetTokensResponse> getTokensResponse = apiInstance.getTokens(confirmationOnly);
            System.out.println(getTokensResponse);

            String burnTokenId = "c1a3078839cd51dfba9e2abe2233b20fe4496854132ba5a9ebc226a13d8278aec7";
            Integer amount = 1000;
            apiInstance.burnToken(burnTokenId, amount);
        } catch (ApiException e) {
            System.err.println("Exception when calling TokenApi#burnToken");
            System.err.println("Status code: " + e.getCode());
            System.err.println("Reason: " + e.getResponseBody());
            System.err.println("Response headers: " + e.getResponseHeaders());
            e.printStackTrace();
        }
    }
}

File: javascript/tokens.js
let TapyrusApi = require('tapyrus_api');
const https = require('node:https');
const http = require('node:http');
const fs = require('node:fs');

let accsessToken = "";
let defaultClient = TapyrusApi.ApiClient.instance;
defaultClient.defaultHeaders = { Authorization: `Bearer ${accsessToken}` }

// You can change the host name and port number
// defaultClient.basePath = 'http://localhost:3000/api/v1'

// Client certificate
if (fs.existsSync('user.p12')) {
  const options = {
    pfx: fs.readFileSync('user.p12'),
    passphrase: '1234'
  };
  httpsAgent = new https.Agent(options);
  defaultClient.requestAgent = httpsAgent;
}

let apiInstance = new TapyrusApi.TokenApi();


// @route   POST /api/v1/tokens/issue 
// @desc    Issue tokens
let issueTokenRequestOpts = {
  'issueTokenRequest': TapyrusApi.IssueTokenRequest.constructFromObject({amount: 200, token_type: 1})
}
apiInstance.issueToken(issueTokenRequestOpts, (error, data, response) => {
  if (error) {
    console.error(error);
  } else {
    console.log(JSON.stringify(data));
  }
});


// @route   GET /api/v1/tokens 
// @desc    Get the list of tokens
let getTokensOpts = {
  'confirmationOnly': true
};
apiInstance.getTokens(getTokensOpts, (error, data, response) => {
  if (error) {
    console.error(error);
  } else {
    console.log(JSON.stringify(data));
  }
});


// @route   POST /api/v1/tokens/:token_id/reissue 
// @desc    Reissue some tokens
let reissueTokenId = "";
let reissueTokenOpts = {
  'reissueTokenRequest': TapyrusApi.ReissueTokenRequest.constructFromObject({amount: 1000})
};
apiInstance.reissueToken(reissueTokenId, reissueTokenOpts, (error, data, response) => {
  if (error) {
    console.error(error);
  } else {
    console.log(JSON.stringify(data));
  }
});

// @route   PUT /api/v1/tokens/:token_id/transfer 
// @desc    Transfer some tokens
let transferTokenId = "c12f7f6e72781c55305572dba9df78e13ee44d0e996f87b0d632db8089c57c0e3d";
let transferTokenRequest = TapyrusApi.TransferTokenRequest.constructFromObject({address: "mnpLDGG9iuyyyRzrdtXPnmugx9Zqr9CZgg", amount: 200})
apiInstance.transferToken(transferTokenId, transferTokenRequest, (error, data, response) => {
  if (error) {
    console.error(error);
  } else {
    console.log(JSON.stringify(data));
  }
});


// @route   DELETE /api/v1/tokens/:token_id/burn 
// @desc    Burn some tokens
let burnTokenId = "c1309cb194cb9e8efaecc3ecaf8e426a0e3303b041ece155981d30872568a9058f"; 
let burnTokenOpts = {
  'amount': 500
};
apiInstance.burnToken(burnTokenId, burnTokenOpts, (error, data, response) => {
  if (error) {
    console.error(error);
  } else {
    console.log('API called successfully.');
  }
});